core/Command/Background/ListCommand.php

```diff
@@ -66,8 +66,12 @@ protected function configure(): void {
 	}
 
 	protected function execute(InputInterface $input, OutputInterface $output): int {
-		$jobs = $this->jobList->getJobsIterator($input->getOption('class'), (int)$input->getOption('limit'), (int)$input->getOption('offset'));
-		$this->writeTableInOutputFormat($input, $output, $this->formatJobs($jobs));
+		$limit = (int)$input->getOption('limit');
+		$jobsInfo = $this->formatJobs($this->jobList->getJobsIterator($input->getOption('class'), $limit, (int)$input->getOption('offset')));
+		$this->writeTableInOutputFormat($input, $output, $jobsInfo);
+		if ($input->getOption('output') === self::OUTPUT_FORMAT_PLAIN && count($jobsInfo) >= $limit) {
+			$output->writeln("\n<comment>Output is currently limited to first " .  $limit . " jobs. Specify `-l, --limit[=LIMIT]` to override.</comment>");
```

Comment:
```suggestion
			$output->writeln("\n<comment>Output is currently limited to " .  $limit . " jobs. Specify `-l, --limit[=LIMIT]` to override.</comment>");
```
Missed it :see_no_evil: